rtengine/gauss.h

```diff
@@ -19,797 +19,12 @@
 #ifndef _GAUSS_H_
 #define _GAUSS_H_
 
-#include <cstdlib>
-#include <cstring>
-#include <cmath>
-#include "opthelper.h"
-#include "stdio.h"
-#include "boxblur.h"
-// classical filtering if the support window is small:
+#include <string.h>
+enum eGaussType {GAUSS_STANDARD, GAUSS_MULT, GAUSS_DIV};
+#define GAUSS_SKIP 0.25
```

Comment:
I don't think these constants need to be in the header, as their usage seems to be limited to `gauss.cc`.  There I would also recommend to use `constexpr auto` instead of `#define` to get proper type checking without overhead.

I am also do not think it is necessary to include `string.h` (or `cstring` for that matter) and `NULL` is probably better replaced by `nullptr` in any case.